Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h

```diff
@@ -252,6 +252,38 @@
     #define LCD_PINS_ENABLE          EXP1_05_PIN
     #define LCD_PINS_D4              EXP1_07_PIN
 
+  #elif ENABLED(WYH_L12864)
+
+  //#error "CAUTION! WYH_L12864 requires wiring modifications. Comment out this line to continue."
```

Comment:
This line shouldn't be commented out by default if you want to alert the user of the required wiring.

Reply:
Thank you! Fixed